tests/sweep_framework/README.md

```diff
@@ -0,0 +1,460 @@
+ OFbZmJABKWg1nzaMM7hU    add    dram    6Fa9k5ABKWg1nzaMCbVI   2024-07-09_18-54-47    PASS      1.0     4e3cc801c
+ OVbZmJABKWg1nzaMM7hV    add    dram    6Va9k5ABKWg1nzaMCbVL   2024-07-09_18-54-47    PASS      1.0     4e3cc801c
+ OlbZmJABKWg1nzaMM7hX    add    dram    6la9k5ABKWg1nzaMCbVN   2024-07-09_18-54-47    PASS      1.0     4e3cc801c
+ O1bZmJABKWg1nzaMM7hb    add    dram    61a9k5ABKWg1nzaMCbVP   2024-07-09_18-54-47    PASS      1.0     4e3cc801c
+ PFbZmJABKWg1nzaMM7he    add    dram    7Fa9k5ABKWg1nzaMCbVR   2024-07-09_18-54-47    PASS      1.0     4e3cc801c
+ PVbZmJABKWg1nzaMM7hg    add    dram    7Va9k5ABKWg1nzaMCbVU   2024-07-09_18-54-47    PASS      1.0     4e3cc801c
+ PlbZmJABKWg1nzaMM7hi    add    dram    7la9k5ABKWg1nzaMCbVW   2024-07-09_18-54-47    PASS      1.0     4e3cc801c
+ P1bZmJABKWg1nzaMM7hk    add    dram    71a9k5ABKWg1nzaMCbVZ   2024-07-09_18-54-47    PASS      1.0     4e3cc801c
+ QFbZmJABKWg1nzaMM7hm    add    dram    8Fa9k5ABKWg1nzaMCbVa   2024-07-09_18-54-48    PASS      1.0     4e3cc801c
+ QVbZmJABKWg1nzaMM7hp    add    dram    8Va9k5ABKWg1nzaMCbVd   2024-07-09_18-54-48    PASS      1.0     4e3cc801c
+ QlbZmJABKWg1nzaMM7hq    add    dram    8la9k5ABKWg1nzaMCbVf   2024-07-09_18-54-48    PASS      1.0     4e3cc801c
+```
+
+```
+$ python3 tests/sweep_framework/query.py --elastic http://172.18.0.2:9200 --module-name add --vector-id 6Va9k5ABKWg1nzaMCbVL vector
+
+{'sweep_name': 'add',
+ 'timestamp': '2024-07-08_19-05-57',
+ 'batch_sizes': '(1,)',
+ 'height': '384',
+ 'width': '4096',
+ 'broadcast': 'w',
+ 'input_a_dtype': 'DataType.BFLOAT16',
+ 'input_b_dtype': 'DataType.BFLOAT16',
+ 'input_a_layout': 'Layout.TILE',
+ 'input_b_layout': 'Layout.TILE',
+ 'input_b_memory_config': {'type': 'tt_lib.tensor.MemoryConfig',
+                           'memory_layout': 'TensorMemoryLayout.INTERLEAVED',
+                           'buffer_type': 'BufferType.DRAM'},
+ 'input_a_memory_config': {'type': 'tt_lib.tensor.MemoryConfig',
+                           'memory_layout': 'TensorMemoryLayout.INTERLEAVED',
+                           'buffer_type': 'BufferType.DRAM'},
+ 'output_memory_config': {'type': 'tt_lib.tensor.MemoryConfig',
+                          'memory_layout': 'TensorMemoryLayout.INTERLEAVED',
+                          'buffer_type': 'BufferType.DRAM'},
+ 'suite_name': 'dram',
+ 'status': 'VectorStatus.VALID'}
+```
+
+```
+$ python3 tests/sweep_framework/query.py --elastic http://172.18.0.2:9200 --module-name add --run-id TlbZmJABKWg1nzaMPbiV result
+
+{'sweep_name': 'add',
+ 'suite_name': 'l1',
+ 'vector_id': '_la9k5ABKWg1nzaMCbWd',
+ 'status': 'TestStatus.NOT_RUN',
+ 'message': 'INVALID VECTOR: Broadcasting along width is not supported for row '
+            'major layout',
+ 'timestamp': '2024-07-09_18-54-51',
+ 'git_hash': '4e3cc801c'}
+```
+
+## Database
+
+Elasticsearch on Apache Lucene database. @Bill/@Raymond Kim to add more details.
```

Comment:
We don't want too many notes on how our internal network topology works. Perhaps put a warning here saying that a common ES cluster is not available on all internal networks, and to slack the infra channel for more clarification.

Reply:
Added a blanket TBD for now because we don't know the configuration yet.